Code/GraphMol/Descriptors/AtomicEnvironmentVector_catch.cpp

```diff
@@ -0,0 +1,292 @@
+        row.push_back(d);
+      }
+      expected_output.push_back(row);
+    }
+
+    for (auto i = 0; i < expected_output.size(); i++) {
+      std::sort(expected_output[i].begin(), expected_output[i].end());
+    }
+
+    std::vector<std::vector<double>> aev_output;
+    RDKit::ROMOL_SPTR mol(RDKit::MolFileToMol(mol_file, true, false));
+    int confId = -1;
+
+    auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(*mol, confId);
+
+    CHECK(aev.rows() == mol->getNumAtoms());
+    CHECK(aev.cols() == 384);
+
+    for (auto i = 0; i < aev.rows(); i++) {
+      std::vector<double> row;
+      for (auto j = 0; j < aev.cols(); j++) {
+        row.push_back(aev(i, j));
+      }
+      aev_output.push_back(row);
+    }
+
+    for (auto i = 0; i < aev_output.size(); i++) {
+      std::sort(aev_output[i].begin(), aev_output[i].end());
+    }
+
+    for (auto i = 0; i < expected_output.size(); i++) {
+      for (auto j = 0; j < expected_output[i].size(); j++) {
+        auto diff = std::fabs(expected_output[i][j] - aev_output[i][j]);
+        CHECK(diff < 0.2);
+      }
+    }
+  }
+
+  SECTION("SO2") {
+    std::string pathName = getenv("RDBASE");
+
+    std::string mol_file =
+        pathName + "/Code/GraphMol/Descriptors/test_data/SO2.mol";
+    std::string fNameSF =
+        pathName + "/Code/GraphMol/Descriptors/test_data/SO2.out";
+
+    std::ifstream instrmSF;
+    instrmSF.open(fNameSF);
+
+    std::string line;
+    std::vector<std::string> tokens;
+    std::vector<std::vector<double>> expected_output;
+
+    while (!instrmSF.eof()) {
+      std::getline(instrmSF, line);
+      tokens = tokenize(line);
+      std::vector<double> row;
+      for (auto v : tokens) {
+        std::istringstream os(v);
+        double d;
+        os >> d;
+        row.push_back(d);
+      }
+      expected_output.push_back(row);
+    }
+
+    for (auto i = 0; i < expected_output.size(); i++) {
+      std::sort(expected_output[i].begin(), expected_output[i].end());
+    }
+
+    std::vector<std::vector<double>> aev_output;
+    RDKit::ROMOL_SPTR mol(RDKit::MolFileToMol(mol_file, true, false));
+    int confId = -1;
+
+    auto aev = RDKit::Descriptors::ANI::AtomicEnvironmentVector(*mol, confId);
+
+    CHECK(aev.rows() == mol->getNumAtoms());
+    CHECK(aev.cols() == 384);
+
+    for (auto i = 0; i < aev.rows(); i++) {
+      std::vector<double> row;
+      for (auto j = 0; j < aev.cols(); j++) {
+        row.push_back(aev(i, j));
+      }
+      aev_output.push_back(row);
+    }
+
+    for (auto i = 0; i < aev_output.size(); i++) {
+      std::sort(aev_output[i].begin(), aev_output[i].end());
+    }
+
+    for (auto i = 0; i < expected_output.size(); i++) {
+      for (auto j = 0; j < expected_output[i].size(); j++) {
+        auto diff = std::fabs(expected_output[i][j] - aev_output[i][j]);
+        CHECK(diff < 0.2);
+      }
+    }
+  }
+
+  SECTION("Ethanol") {
+    std::string pathName = getenv("RDBASE");
+
+    std::string mol_file =
```

Comment:
From my reading of the rdkit's source it seems they prefer camelCase vs snake_case in variable names in the C++ code.

Reply:
I had made the mistake in other files as well. I've fixed it now